src/sentry/static/sentry/app/components/errorRobot.jsx

```diff
@@ -24,19 +21,20 @@ const ErrorRobot = createReactClass({
     sampleIssueId: PropTypes.string,
 
     gradient: PropTypes.bool,
-  },
+  };
 
-  getInitialState() {
-    return {
+  constructor(props) {
```

Comment:
Should we use class props for state and remove the constructor?

Reply:
The `sampleIssueId` state value is based on the prop value though.

Reply:
It'll work if you define it as

```javascript
    state = {
      error: false,
      loading: false,
      sampleIssueId: this.props.sampleIssueId,
    };
```